addons/survey/views/survey_survey_views.xml

```diff
@@ -195,6 +194,19 @@
             </form>
         </field>
     </record>
+
+    <record id="action_print_survey" model="ir.actions.server">
```

Comment:
Can we make a separate file "ir_actions_server_data.xml" for this? Seems to be the most used convention.